assets/js/barChart.js

```diff
@@ -50,7 +50,7 @@ class BarChart {
 
         // Create scaling for y that converts formattedData names to pixels
         let yScale = d3.scale.ordinal()
-            .domain(formattedData.map(function(d) {return d.name; }))
+            .domain(formattedData.map(function(d) {return d.trimName; }))
```

Comment:
I think we want to use the full name here. Consider the case where two names only differ on character 51 and we truncate on character 50.

```
   [
       "some...long...name1"
       "some...long...name2"
   ]
```

After truncation we would get:

 ```
   [
       "some...long...name"
       "some...long...name"
   ]
```
Each bar in the chart needs a distinct name in the y-axis "domain" in order to be drawn at a distinct height along the y-axis. In the case above, there would be no way for the y axis to distinguish the names, and you would probably get overlapping bars at that same position instead of two different bars at different heights.

It looks like one possible solution might use be to use the d3 axis's `.tickFormat()` to set the names of the labels different from the domain of the y-axis.